README.md

```diff
@@ -43,13 +43,19 @@ cookiecutter gh:ssciwr/cookiecutter-cpp-project
 ```
 
 This will start an interactive prompt that will configure and generate your project.
+One of the prompts will ask you for a remote repository URL, so you should head to
+the Git hosting service of your choice (e.g. `github.com`) and add a new repository.
```

Comment:
some more detail would be good here, e.g.
- link directly to https://github.com/new
- user should create a new *empty* repo? or doesn't matter if they click the 'add readme' option?
- which url: ssh or https?

Reply:
I tried it locally using `git@github.com:lkeegan/temp-test.git` for the url and got this error message
```
Traceback (most recent call last):
  File "/tmp/tmpabf2anc2.py", line 69, in <module>
    repo.add_submodule("https://github.com/pybind/pybind11.git", "ext/pybind11", tag="v2.6.1")
  File "/tmp/tmpabf2anc2.py", line 36, in add_submodule
    subprocess.check_call("git remote add origin git@github.com:lkeegan/temp-test.git".split())
  File "/home/lkeegan/.pyenv/versions/3.8.6/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['git', 'remote', 'add', 'origin', 'git@github.com:lkeegan/temp-test.git']' returned non-zero exit status 128.
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
```

Reply:
I can reproduce this, but only when the `python_bindings` config is `Yes`. Will investigate.

Reply:
Oh, this was just a supersilly bug that was only triggered in the presence of two submodules.

Reply:
Linked the page. Mentioned that the repo should be empty - otherwise your first push will be required to be forced. ssh vs https is irrelevant.